Makefile.doc

```diff
@@ -31,8 +31,8 @@ DVIPS:=dvips
 HTMLSTYLE:=coqremote
 
 # Sphinx-related variables
-OSNAME:=$(shell uname -o)
-ifeq ($(OSNAME),Cygwin)
+OSNAME:=$(shell uname -s)
+ifeq ($(findstring CYGWIN,$(OSNAME)),CYGWIN)
```

Comment:
Why aren't we getting this info from configure?

Reply:
Good remark! configuration already gives the arch, it could give the operating system too.

(In any case, I'm not going to necessarily follow this thread regularly, so tell me privately if I eventually need to do something in this direction.)